### PR TITLE
 [MIG] website_cookie_notice: Migrated to 11.0

### PR DESCRIPTION
**Pull request by njeudy**
Depends on:

- [x] website_legal_page: #403

### CODE REVIEW

**Comment by njeudy** [comment]
Will only migratie cookie_notice in this because legal page already migrate In #403

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by pedrobaeza** [comment]
OK, I see it's because dependencies. Adding on the main comment.

**Comment by njeudy** [comment]
Yes that's why I migrated both in same merge request :) but someone did it .. I will try to help review #403 :)

File: website_cookie_notice/__manifest__.py
Comment (simahawk) [inline website_cookie_notice/__manifest__.py]: 11

File: website_cookie_notice/__manifest__.py
Comment (simahawk) [inline website_cookie_notice/__manifest__.py]: https://github.com/OCA/website

File: website_cookie_notice/__manifest__.py
Comment (simahawk) [inline website_cookie_notice/__manifest__.py]: you can drop this key

File: website_cookie_notice/controllers/__init__.py
Comment (simahawk) [inline website_cookie_notice/controllers/__init__.py]: remove coding headers from *.py

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
for the build: you can add oca_dependencies.txt with the branch where you've migrated ws legal page.

**Comment by njeudy** [comment]
@simahawk I try to add oca_dependency.txt but:

fatal: Remote branch refs/pull/408/head not found in upstream origin
Traceback (most recent call last):
  File "/home/travis/maintainer-quality-tools/travis/clone_oca_dependencies", line 125, in <module>
    run(deps_checkout_dir, build_dir)
  File "/home/travis/maintainer-quality-tools/travis/clone_oca_dependencies", line 99, in run
    url, branch)
  File "/home/travis/maintainer-quality-tools/travis/clone_oca_dependencies", line 67, in git_checkout
    subprocess.check_call(command)
  File "/opt/python/3.5.4/lib/python3.5/subprocess.py", line 271, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'clone', '-q', 'https://github.com/OCA/website', '-b', 'refs/pull/408/head', '--single-branch', '--depth=1', '/home/travis/dependencies/website']' returned non-zero exit status 128

Think I should wait for legal_page to be merged.

**Comment by simahawk** [comment]
@njeudy the file is wrong :) you should provide the URL to your fork an put your branch, see https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt
Also, it's better to put temporary oca_dependencies.txt changes in a separated commit, like "DROP_ME_BEFORE_MERGE include pending PR" so that it's easier to spot it before merge and to roll it back. :wink:

**Comment by njeudy** [comment]
@simahawk  Ok thanks

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@eLBati @Yajo ping :)

**Comment by yajo** [comment]
I'm waiting for #403

**Comment by pedrobaeza** [comment]
Please rebase now that the dependency is merged.

**Review by yajo — APPROVED** [review]

**Comment by njeudy** [comment]
@simahawk @Yajo can it be merged ?

**Comment by pedrobaeza** [comment]
Yes, merging

**Pull request merged**
